ramls/request.json

```
@@ -101,6 +101,16 @@
               "type": "string",
               "description": "The name of the location",
               "readonly": true
+            },
+            "LibraryName": {
```

JSON property names should start with a lower case letter.
```suggestion
            "libraryName": {
```